File: EXAM/ullcu/u1/Exam 27 and 33/soccerBall.js
export default class soccerBall {
    constructor(id, brand, size, material){
        this._id = id;
        this._brand = brand;
        this._size = size;
        this._material = material;
    }

toString(){
    return `soccerBall{id='${this._id}', brand='${this._brand}', size='${this.size}}', material=${this._material}}`;
}

get id() {
    return this._id;
}

set id(value) {
    this._id = value;
}

get brand() {
    return this._brand;
}

set brand(value) {
    this._brand = value;
}

get size() {
    return this.size;
}

set size(value) {
    this.size = value;
}

get material() {
    return this.material;
}

set material(value) {
    this.material = value;
}
}

File: hw/ullcu/U2/HW_17_CreateAnObjectFile/src/SoccerBall.java

/**
 *
 * @author USUARIO
 */
import java.io.Serializable;

public class SoccerBall implements Serializable {
    private int id;
    private String brand;
    private double price;
    private String Material;

    public SoccerBall(int id, String brand, double price, String Material) {
        this.id = id;
        this.brand = brand;
        this.price = price;
        this.Material = Material;
    }

    @Override
    public String toString() {
        return "SoccerBall{" + "id=" + id + ", brand=" + brand + ", price=" + price + ", Material=" + Material + '}';
    }


   
    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the brand
     */
    public String getBrand() {
        return brand;
    }

    /**
     * @param brand the brand to set
     */
    public void setBrand(String brand) {
        this.brand = brand;
    }

    /**
     * @return the price
     */
    public double getPrice() {
        return price;
    }

    /**
     * @param price the price to set
     */
    public void setPrice(double price) {
        this.price = price;
    }

    /**
     * @return the Material
     */
    public String getMaterial() {
        return Material;
    }

    /**
     * @param Material the Material to set
     */
    public void setMaterial(String Material) {
        this.Material = Material;
    }
}
